crates/wasm-mutate/src/mutators/peephole/dfg.rs

```diff
@@ -1126,6 +1134,19 @@ impl<'a> DFGBuilder {
         self.push_node(op([Id::from(rightidx), Id::from(leftidx)]), idx);
     }
 
+    fn ternop(&mut self, idx: usize, op: fn([Id; 3]) -> Lang) {
+        let a = self.pop_operand(idx, false);
+        let b = self.pop_operand(idx, false);
+        let c = self.pop_operand(idx, false);
+
+        // The operands should not be the same.
+        assert_ne!(a, b);
+        assert_ne!(a, c);
+        assert_ne!(b, c);
+
+        self.push_node(op([Id::from(a), Id::from(b), Id::from(c)]), idx);
```

Comment:
I think this order may need to be reversed (judging by the other methods that do this)